manifests/operator.yaml

```diff
@@ -2572,7 +4656,11 @@ spec:
                       - Unknown
                       type: string
                     type:
-                      description: type of condition in CamelCase or in foo.example.com/CamelCase. --- Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be useful (see .node.status.conditions), the ability to deconflict is important. The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
+                      description: type of condition in CamelCase or in foo.example.com/CamelCase.
+                        --- Many .condition.type values are consistent across resources
+                        like Available, but because arbitrary conditions can be useful
+                        (see .node.status.conditions), the ability to deconflict is
+                        important. The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
```

Comment:
It would be nice (though, I think it's already too late) if all these 1000s of reformatted lines were put into a separate "refactoring-only" commit. Otherwise, it's really hard to see what's really changed versus rafactored.

Reply:
@iagaponenko , this file is automatically generated, and should be available via a webserver instead of git, but I am not sure we can do that currently (@ktlim would you please have any idea about it?)